From 845bdab38ec8bac53d570a25a39598f797dc4ea6 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 23 Nov 2018 20:31:18 +0900
Subject: [PATCH 528/909] arm64: dts: r8a77965: Add cpg reset for LVDS
 Interface

commit 21d3e7a49be3e40894f2c264b90d24cc4f995fd3 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

It is necessary to reset the LVDS Interface according to display on/off.
Therefore, this patch adds CPG reset properties in DU device node
for the R8A77965 SoC.

Reported-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
[takeshi.kihara.df: rebased]
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77965.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77965.dtsi b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
index 48c02e445c23..f86a79a0b781 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
@@ -2143,6 +2143,10 @@
 				 <&cpg CPG_MOD 723>,
 				 <&cpg CPG_MOD 721>;
 			clock-names = "du.0", "du.1", "du.3";
+			resets = <&cpg 724>,
+				 <&cpg 724>,
+				 <&cpg 722>;
+			reset-names = "du.0", "du.1", "du.3";
 			status = "disabled";
 
 			vsps = <&vspd0 0 &vspd1 0 &vspd0 1>;
-- 
2.17.1

